app/labor/rate_limit_checker.rb

```diff
@@ -22,16 +22,4 @@ def limit_by_email_recipient_address(address)
     EmailMessage.where(to: address).
       where("sent_at > ?", 2.minutes.ago).size > 5
   end
-
-  def ping_admins
```

Comment:
It would be better to keep the `ping_admins` method and add a `ping_admins_without_delay` method in which to call `Labor::RateLimitCheckerJob.perform_now(user.id)`. Check this comment https://github.com/thepracticaldev/dev.to/issues/2497#issuecomment-485396688

Reply:
@rhymes yes, I went through that comment. But `ping_admins` is not being used anywhere else so changed it this way. Anyways, I will follow what you have said.

Reply:
@rhymes I got what you are saying now. Did not read it carefully enough the first time.